app/scss/palm-theme.scss

```diff
@@ -0,0 +1,48 @@
+// Colors
+$green_light:					#456431;
+$green: 						#2C5114;
+$green_dark: 					#172A0B;
+$green_darkest:		        	#313930;
+
+$cyan_very_light:               #BDC8CB;
+$cyan_light:		            #A9B8BC;
+$cyan_grey:		            	#CBD4D7;
+$cyan_dark_grey:             	#819D9C;
+$cyan_dark_desaturated: 		#5A7479;
+$cyan_dark:                     #124559;
+
+$grey_dull: 					#C8C7C2;
+
+$white_egg:                     #F7FAEF;
+$white_alabaster:		        #F7F7F7;
+
+$blue_light:                    #87F0FB;
+$blue_dark_grayish:             #70888F;
+
+// Theme variables
+$background:              		url("../images/panel/leaf.svg");
+
+$top_bar:						$green;
+
+$summary_text:					#FFF;
+$caption:             			$white_alabaster;
+$title_text:					$cyan_grey;
+
+$enabled_feature:				$blue_light;
+$bright_enabled_feature:		$white_egg;
+$paled_enabled_feature:			$cyan_dark_grey;
+
+$disabled_feature:             	$blue_dark_grayish;
+$bright_disabled_feature:		$cyan_very_light;
+$paled_disabled_feature:		$cyan_light;
+
+$disabled_cliqz_feature:		#FFF;
+$inactive_cliqz_feature: 		$grey_dull;
+
+$tab_inactive:      			$green_light;
+$tab_active: 					$green_darkest;
+
+$tracker-list: 					$green_dark;
+$divider: 						$cyan_dark_desaturated;
+
+@import './themes/theme.scss';
```

Comment:
I am poking around to see if we can use the preferred `@use` directive instead (https://sass-lang.com/documentation/at-rules/use). It's not working yet but we should look into it some more....or have you already tried and it didn't work?

Reply:
Oh, I didn't even know @use existed. I tried replacing @import with @use inside the account project (where I moved this code) and it seems to work there!